files/image_config/kubernetes/scripts/kube_reset.sh

```diff
@@ -0,0 +1,3 @@
+kubeadm reset -f
+rm -rf /etc/cni/net.d
```

Comment:
why?

Reply:
Because 'kubeadm reset" does not remove this dir and it explicitly tell the caller to remove it. If you don't subsequent join would fail.